src/main/java/com/labsynch/labseer/domain/Lot.java

```diff
@@ -614,7 +614,7 @@ public static TypedQuery<LotsByProjectDTO> findLotsByProjectsList(List<String> p
 
         EntityManager em = Lot.entityManager();
         String query = "SELECT new com.labsynch.labseer.dto.LotsByProjectDTO( "
-                + "lt.id as id, lt.corpName as lotCorpName, lt.lotNumber as lotNumber, lt.registrationDate as registrationDate, prnt.corpName as parentCorpName, lt.project as project) "
+                + "lt.id as id, lt.corpName as lotCorpName, lt.lotNumber as lotNumber, lt.registrationDate as registrationDate, prnt.corpName as parentCorpName, lt.project as project, lt.chemist as chemist) "
```

Comment:
@hhan-schrodinger Per you question on the main thread. You should be able to join to vendor here.

e.g.
```
SELECT SELECT new com.labsynch.labseer.dto.LotsByProjectDTO(....etc. , vnd as vendor)
```

And:
```
+ "JOIN lt.vendor vnd "
```

Then you need to add vendor to the LotsByProjectDTO (initialization, getters and setters).

Reply:
@brianbolt I didn't realize it was a link to the vendor table, thanks!